src/store/utils.js

```diff
@@ -50,3 +50,10 @@ export function downloadFileInBrowser(filename, content, type) {
         document.body.removeChild(link);
     }
 }
+
+export function stxtap(el, func) {
+    if (el) {
+        CIQ.installTapEvent(el);
```

Comment:
What is `installTapEvent` really doing? Can we do it ourself or is it dependent on the internals of the chartiq librarry?

Reply:
@aminroosta ah shucks. The `ref` is called multiple times. Will have to add a check for that. Will ping you when I'm done

I actually add `installTapEvent` because without it `stxtap` event doesn't get fired. This wasn't the case before (28 Mar) so I wasn't sure what happen. In the docs (http://documentation.chartiq.com/CIQ.html#.installTapEvent) it writes: 

_Converts an object to emit "stxtap" events. This uses CIQ#safeClickTouch. You should use addEventListener("tap") to receive the events._

Of course, "tap" doesn't work. We need to use "stxtap"